src/types/App/index.d.ts

```diff
@@ -198,7 +207,8 @@ declare namespace State {
     blockListState: BlockListState
     transactionState: TransactionState
     statistics: Statistics
-    statisticsChartDatas: StatisticsChartDatas
+    statisticsChartDatas: StatisticsChartData[]
```

Comment:
`Datas` is invalid, please use `DataList` or something like that.

Reply:
Oh, there are lots of place to be updated.

Reply:
It's easy to understand, we can update it later or never.

Reply:
OK. I will refactor later.